### ISSUE TITLE
sides of footer

### ISSUE THREAD

**Issue by johnmarriott** [BSteffaniak/MikeMcDannaldLab#3]
fill in the sides of the footer (similar to #2)

also set the page background to the footer bottom color -- if you swipe-scroll the bottom of the page you see a white band under it

**Comment by BSteffaniak** [comment]
I don't see any white under it. What are you looking at it on?

**Comment by johnmarriott** [comment]
Here's what I get, this is when finger-swiping to scroll up.  The white space below the footer is in the browser window.

<img width="978" alt="whitespace" src="https://cloud.githubusercontent.com/assets/2177654/23046731/f6bc19e0-f470-11e6-8f64-1711905a23f2.png">

**Comment by BSteffaniak** [comment]
Oh you mean when stretching the page?

**Comment by johnmarriott** [comment]
Yes, scrolling past (with a gesture) what a window's scroll bar would let you.

**Comment by BSteffaniak** [comment]
Ok. The footer is not fixed, so setting the background color would set the whole page's background. I'd have to manually put in some divs on left and right to cover the black if I were to change the page's background color.

**Comment by BSteffaniak** [comment]
Do you have any ideas on fixing it more simply that what I just suggested?

**Comment by johnmarriott** [comment]
No I think (if it's worth fixing) that's how to fix it.  The left/right divs wouldn't be visible on xs/sm (?)... something like [xs-hidden,md-2][xs-12,md-8][xs-hidden,md-2] for those three sections (translate that from bootstrap language to this).

**Comment by BSteffaniak** [comment]
Well hold on. If I put div overlays on left and right and you stretch to left or right, you would see black.

**Comment by johnmarriott** [comment]
Can you make the footer a box?  E.g. https://arstechnica.com would suffer this same problem but adds a small padding below the footer.
